cristin-commons/src/main/java/no/unit/nva/cristin/model/Constants.java

```diff
@@ -61,6 +61,7 @@ public class Constants {
     public static final String POSITION = "position";
     public static final String HASHTAG = "#";
     public static final String SLASH_DELIMITER = "/";
+    public static final String CRISTIN_INSTITUTION_HEADER = "Cristin-Institution";
```

Comment:
We should indicate what the value means in context, so possibly something like "On-Behalf-Of-Institution", if possible to change this (it gives more meaning in context and says what it does).

On a side note, X- prefixes, which I half expected to see here, were deprecated in 2012…the things you learn doing PRs.